File: beginner/javascript/1837.js
var input = require('fs').readFileSync('stdin', 'utf8');
var linhas = input.split(' ');

var a, b, c, d, e, f = 0, q, r;

a = parseInt(linhas[0]);
b = parseInt(linhas[1]);

if(a < 0){

    e = b;

    if(b < 0) 
        e = b *- 1;

    for(r = 0; r < e; r++){
        f = a - r;
        if(f % b== 0) 
            break;
    }
    q = f / b;
}else{
    q = a / b;
    r = a % b;
}

console.log(`${q} ${r}`);

File: beginner/python/1837.py
a, b = list(map(int, input().split()))

if a < 0:

    e = b

    if(b < 0):
        e = b *- 1

    for r in range(0,e):
        f = a - r
        if(f % b == 0):
            break
    
    q = f / b
else:
    q = a / b
    r = a % b

print("%d %d" % (q, r))